### PR TITLE
devmapper: initialize log levels

### PR DESCRIPTION
**Pull request by vbatts**
I'll say, the importance of this change is that the skipping debug level log did the opposite of what we would want. In that when docker is run in debug mode, then _no_ logging would occur.

Additionally, there are tell-tale output from device-mapper on WARN level 4, so this makes that the default level.

Signed-off-by: Vincent Batts vbatts@redhat.com

### CODE REVIEW

**Comment by SvenDowideit** [comment]
oh go on, dare you to tell me that these magic settings are documented in your readme. :D

**Comment by vbatts** [comment]
Which magic settings? Setting DEBUG? That is a known thing and when you pass the `-D` flag to client or deamon it sets the DEBUG env as well.

**Comment by vbatts** [comment]
made the levels more clear. PTAL.

File: daemon/graphdriver/devmapper/deviceset.go [hunk 1]
```diff
@@ -939,6 +941,11 @@ func (devices *DeviceSet) closeTransaction() error {
 }
 
 func (devices *DeviceSet) initDevmapper(doInit bool) error {
+	if len(os.Getenv("DEBUG")) > 0 {
```
Comment (LK4D4) [inline daemon/graphdriver/devmapper/deviceset.go]: Actually everywhere in our code we using `os.Getenv("DEBUG") != "` :)

**Comment by vbatts** [comment]
@LK4D4 changed. PTAL.

File: pkg/devicemapper/log.go [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package devicemapper
+
+// definitions from lvm2 lib/log/log.h
+const (
+	_              = iota
```
Comment (LK4D4) [inline pkg/devicemapper/log.go]: What is this?

Comment (vbatts) [inline pkg/devicemapper/log.go]: iota starts at 0, which we don't need. But this shows me that we need it to start at 2 and go to 7. I'll make this clearer.

File: pkg/devicemapper/log.go [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package devicemapper
+
+// definitions from lvm2 lib/log/log.h
+const (
+	LogLevelFatal  = 2 + iota // _LOG_FATAL
```
Comment (vbatts) [inline pkg/devicemapper/log.go]: @LK4D4 Is this clearer?

Comment (LK4D4) [inline pkg/devicemapper/log.go]: Yup, also this is different logic as I see :) So we fixed bug!

**Comment by LK4D4** [comment]
LGTM

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**